doc/userguide/rules/rules-internals.rst

```diff
@@ -0,0 +1,230 @@
+Rules Processing
```

Comment:
"Rule Processing"?

Or is this intentional.

Reply:
Maybe a non-native slip, as the engine processes rules. What sounds more natural?